Assets/Scripts/BlockBehaviour.cs

```diff
@@ -86,29 +93,44 @@ public bool MoveBlock(BlockFace face)
             return false;
         }
 
+        if (_isFalling)
+        {
+            UpdateSpeed();
+        }
+
         bool hit = _blockFaceBehaviour.FireRaycastFromFace(SkinToLengthRatio, CollidableLayers, face);
         if (!hit)
         {
             _targetPosition = transform.position + (face.GetNormal() * _blockFaceBehaviour.GetFaceLength());
             _isTranslating = true;
+            _isFalling = true;

```

Comment:
This isnt necessarily true? If a block moves on top of another block, it wouldnt be falling.